src/NServiceBus.Core/Encryption/Encryptor.cs

```diff
@@ -51,7 +51,7 @@ bool VerifyPrerequisite(FeatureConfigurationContext context)
         static List<PropertyInfo> GetEncryptedProperties(FeatureConfigurationContext context)
         {
             var conventions = context.Settings.Get<Conventions>();
-            return context.Settings.GetAvailableTypes()
+            return context.Settings.GetAvailableMessageTypes()
```

Comment:
Looking more how the encrypted properties work, it seems like this was a place that should have been using `IsMessageType` but wasn't, so this does seem like a change that's desirable.

:+1:  for keeping the change.